tornjak-frontend/src/components/navbar.tsx

```diff
@@ -70,6 +122,43 @@ class NavigationBar extends Component<NavigationBarProp, NavigationBarState> {
                 }}
               >Tornjak Dashboard</a>
             </div>
+            <div className='header-toolbar'>
```

Comment:
this entire section probably should be a class on its own

Reply:
done!